app/controllers/facilitator/projects_controller.rb

```diff
@@ -44,11 +44,11 @@ def find_facilitator
   end
 
   def project_creation_params
-    params.require(:project_creation_form).permit(project_attributes: [:name, :location, :start_on, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
+    params.require(:project_creation_form).permit(project_attributes: [:name, :location, :start_on, :end_on, :status, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
   end
 
   def project_modification_params
-    params.require(:project_modification_form).permit(project_attributes: [:name, :location, :start_on, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
+    params.require(:project_modification_form).permit(project_attributes: [:name, :location, :start_on, :end_on, :status, students_attributes: [:id, :name, :class_name, :birthday, :comments, :_destroy]])
```

Comment:
Hmm, at the moment, `:status` and `:end_on` aren't exposed in the form views. But I wonder if they should for the update form. A user should be able to extend the duration of the project.